deel-3-webpagina/styles/styles.css

```diff
@@ -1 +1,108 @@
-/*Schrijf hier jouw css*/
\ No newline at end of file
+/*Schrijf hier jouw css*/
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Fredoka+One&display=swap');
+/*
+beige: F2F3D9
+orange: DC9E82
+Red: C16E70
+oxford blue: 030027
+space cadet: 151E3F
```

Comment:
Al deze kleuren had je in je `:root { } ` kunnen gooien en dan hier ze een (var) naam geven en ze aanroepen door heel je file. Hoofdstuk 8 of 9 van frontend. Wel heel intresant.